lib/airbrake.js

```diff
@@ -10,6 +10,7 @@ var stackTrace = require('stack-trace');
 var hashish = require('hashish');
 var querystring = require('querystring');
 var stringify = require('json-stringify-safe');
+var execSync = require('sync-exec');
```

Comment:
Why add a dependency when you can use [`child_process.execSync`](https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options)